node_modules/oae-core/userdetails/js/userdetails.js

```diff
@@ -0,0 +1,161 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['jquery', 'oae.core'], function($, oae) {
+
+    return function(uid) {
+
+        // The widget container
+        var $rootel = $('#' + uid);
+
+        /**
+         * Edit the profile
+         */
+        var editUserDetails = function() {
+            // Disable the form
+            $('#userdetails-form *', $rootel).prop('disabled', true);
```

Comment:
Can we just disable the input / button elements instead of `*`? To avoid duplication, we can do `$toDisable = $('#userdetails-form input, #userdetails-form button')`, and then use that object for both the enabling and disabling.